encoder/vaapiencoder_vp9.cpp

```diff
@@ -257,10 +257,10 @@ bool VaapiEncoderVP9::fill(VAEncPictureParameterBufferVP9* picParam,
         }
     }
 
-    picParam->frame_width_src = width();
-    picParam->frame_height_src = height();
-    picParam->frame_width_dst = width();
-    picParam->frame_height_dst = height();
+    picParam->frame_width_src = ALIGN8(width());
```

Comment:
Seems need driver to fix it.  frame_height_dst and frame_width_dst  are used to scale not clip. We can create a bug to driver, and close this issue.